src/AWS.Deploy.CLI/AWS.Deploy.CLI.csproj

```diff
@@ -8,10 +8,10 @@
     <PackAsTool>true</PackAsTool>
     <Description>Opinionated tooling that simplifies deployment of .NET applications with minimum AWS knowledge.</Description>
     <Product>AWS .NET deployment tool</Product>
-    <PackageId>AWS.Deploy.CLI</PackageId>
+    <PackageId>AWS.Deploy.Tools</PackageId>
     <PackageTags>AWS;Amazon;ElasticBeanstalk;ECS;Deploy</PackageTags>
-    <AssemblyName>AWS.Deploy.CLI</AssemblyName>
-    <RootNamespace>AWS.Deploy.CLI</RootNamespace>
+    <AssemblyName>AWS.Deploy.Tools</AssemblyName>
```

Comment:
We don't need to change the `AssemblyName` and `RootNamespace`. In particular we don't want to change the `RootNamespace` since that would make it different then all of the `cs` files currently in the project.

Reply:
Fixed